data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/s3/SqsWorker.java

```diff
@@ -336,6 +343,27 @@ private S3ObjectReference populateS3Reference(final String bucketName, final Str
                 .build();
     }
 
+    private ExpiryItem getExpiryItem(final Message message, final Instant expirationTime, final AcknowledgementSet acknowledgementSet) {
```

Comment:
We should probably make this feature configurable.

Reply:
Agreed, was planning to add this